packages/calendar/core/src/helpers/timezone.ts

```diff
@@ -0,0 +1,18 @@
+import { format } from "date-fns-tz";
+import { enUS } from "date-fns/locale";
+
+export const getPrimaryTimeZone = (timeZones: string[]) => {
+  return timeZones[1] ?? timeZones[0] ?? "UTC";
+};
+
+export const getTimeZoneAbv = (timeZone: string, date = new Date()) => {
+  const title = format(date, "zzz", { timeZone });
+
+  const convertedTimeZones = ["GMT-4", "GMT-5", "GMT-6", "GMT-7", "GMT-8"];
+  if (convertedTimeZones.includes(title)) {
+    // Forcefully use enUS locale
```

Comment:
```suggestion
    // Forcefully use enUS locale for convertedTimeZones
```

Reply:
@bgavas to bring out conversation here
> Richard: So I understand this is a temporary fix valid for the next 6 months then?
> Burak Gavas: It is not a temporary fix. We forcefully use enUS locale if time zone is ‘GMT-4’, ‘GMT-5’, ‘GMT-6’, ‘GMT-7’, ‘GMT-8’

I'm a bit surprised by that because these timezones apply to multiple locales, not only en-US.
What's the reason we need to force this to be en-US?

I don't understand this from the comment in the code. 🤔 
<img width="919" alt="image" src="https://user-images.githubusercontent.com/18185649/201475818-a126e32f-c458-4804-941d-82d70df69625.png">